dune-project

```
@@ -0,0 +1,5 @@
+(lang dune 3.6)
```

```suggestion
(lang dune 3.7)
```
so that the Dune CI is happy.